languages/CSS.md

```diff
@@ -67,5 +67,13 @@
 
 [**Tailwind CSS**](https://github.com/tailwindcss/tailwindcss) — A utility-first CSS framework for rapid UI development.
 
+---
+
+## U
+
+[**UIkit**](https://getuikit.com/) — A lightweight and modular front-end framework
```

Comment:
Please, change link to the GitHub repository.

Reply:
Would do so immediately.